lib/dart_sass.ex

```diff
@@ -172,9 +168,17 @@ defmodule DartSass do
       stderr_to_stdout: true
     ]
 
-    {sass_path, args} = sass(args ++ extra_args)
+    {path, args} = sass(args ++ extra_args)
 
-    sass_path
+    # TODO: Remove when dart-sass will exit when stdin is closed.
+    # Link: https://github.com/sass/dart-sass/pull/1411
```

Comment:
I think you only need this if the --watch flag (or whatever dart-sass uses) is given.

Reply:
😬 of course